tests/drivers/build_all/sensor/Kconfig

```
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config RUN_GENERIC_SENSOR_TEST
```

```suggestion
config GENERIC_SENSOR_TEST
```
The "run" verb seems redundant.